autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/SectionSelect.tsx

```diff
@@ -186,7 +182,9 @@ const SectionSelect: React.FC<SectionSelectProps> = ({ id }): JSX.Element => {
             />
           </ListItemIcon>
           <ListItemText disableTypography>
-            {meetingRows}
+            <table className={styles.sectionDetailsTable}>
+              {meetingRows}
+            </table>
```

Comment:
This gives a console error:
![image](https://user-images.githubusercontent.com/28655462/90984712-5c721400-e53c-11ea-8584-328e8e5cb0af.png)
You should be able to just wrap `meetingRows` in a `tbody` to fix it